utils.py

```diff
@@ -5,6 +5,10 @@
 def extract_gs_file(folder):
     """Extract goldstandard file from folder."""
     files = glob(os.path.join(folder, "*"))
+
+    # Filter out the manifest file
+    files = [f for f in files if os.path.basename(f) != 'SYNAPSE_METADATA_MANIFEST.tsv']
+
```

Comment:
Alternatively, we could exclude the file when we first use `globl`? e.g.

```suggestion
    # Get all files from the folder, excluding the SYNAPSE_METADATA_MANIFEST.tsv file.
    files = glob(os.path.join(
        folder,
        "[!SYNAPSE_METADATA_MANIFEST.tsv]*"
    ))
```

Reply:
Ah no, nevermind! I forgot `glob` uses Unix expansion rules, not regex, to do "pattern" matching.

Ignore this suggestion :-)